packages/eslint-config-base/index.js

```diff
@@ -167,6 +167,10 @@ module.exports = {
             element: true,
           }
         },
+        whitelist: {
+          attr: true
+        },
+        checkProperties: true,
```

Comment:
I wonder how much errors this will throw on options objects for third party libraries :)

Is there we way we can only check the "declaration side" of those objects?

Reply:
Discussion about why they changed the default to `false`: https://github.com/sindresorhus/eslint-plugin-unicorn/issues/271

Reply:
> Is there we way we can only check the "declaration side" of those objects?
don't think that's possible.

Personally, I wouldn't enable this because of the potential errors for third party libs.

Reply:
I tried it in a big project and the only thing I ran into was the `attr`. That's why I added it in the `whitelist`. I would suggest we keep a close eye on this and add more things in the `whitelist` if needed.